test/e2e/specs-jetpack-calypso/wp-jetpack-plans-spec.js

```diff
@@ -68,13 +70,13 @@ describe( `[${ host }] Jetpack Plans: (${ screenSize }) @jetpack`, function () {
 			return await jetpackDashboard.clickUpgradeNudge();
 		} );
 
-		step( 'Can click upgrade on Jetpack landing page', async function () {
-			const searchLandingPage = await JetpackComSearchLandingPage.Expect( driver );
-			return await searchLandingPage.upgrade();
-		} );
-
-		step( 'Can then see secure payment component', async function () {
-			return await SecurePaymentComponent.Expect( driver );
```

Comment:
This upgrade link no longer redirects users to Jetpack Search Landing page, instead, it redirects users directly to the checkout page with Jetpack Search in the cart.